docs/env-variables.md

```diff
@@ -31,7 +31,8 @@
 | `--git-ssh-private-key-path` | `ENVBUILDER_GIT_SSH_PRIVATE_KEY_PATH` |  | Path to an SSH private key to be used for Git authentication. If this is set, then GIT_SSH_PRIVATE_KEY_BASE64 cannot be set. |
 | `--git-ssh-private-key-base64` | `ENVBUILDER_GIT_SSH_PRIVATE_KEY_BASE64` |  | Base64 encoded SSH private key to be used for Git authentication. If this is set, then GIT_SSH_PRIVATE_KEY_PATH cannot be set. |
 | `--git-http-proxy-url` | `ENVBUILDER_GIT_HTTP_PROXY_URL` |  | The URL for the HTTP proxy. This is optional. |
-| `--workspace-folder` | `ENVBUILDER_WORKSPACE_FOLDER` |  | The path to the workspace folder that will be built. This is optional. |
+| `--workspace-folder` | `ENVBUILDER_WORKSPACE_FOLDER` |  | The path to the workspace folder that will be built. This is optional. Defaults to `[workspaces folder]/[name]` where name is the name of the repository or `empty`. |
+| `--workspaces-folder` | `ENVBUILDER_WORKSPACES_FOLDER` | `/workspaces` | The path under which workspaces will be placed when workspace folder option is not given. |
```

Comment:
I realize that this is a terrible time to suggest a different name, but would `ENVBUILDER_WORKSPACE_BASEDIR` or similar differentiate it more significantly from `ENVBUILDER_WORKSPACE_FOLDER`? Just differentiating two parameters by a single character is going to be super annoying, I feel.

Reply:
I agree about the similarity. I'll use your suggestion (although s/BASEDIR/BASE_DIR/ to match our other envs) 👍🏻.